gui_qt/template_generation_report.py

```
@@ -0,0 +1,119 @@
+                values[match.group(1)] = match.group(2).strip()
+            continue
+        if in_section and not line.startswith("- "):
+            break
+    return values
+
+
+def running_template_generation_summary() -> TemplateGenerationSummary:
+    return TemplateGenerationSummary(
+        status="running",
+        heading="正在生成翻译模板",
+        message="正在准备 work/game 并调用 Ren'Py 生成翻译模板，请稍候。",
```

Ruff (0.15.20)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)